src/OrbitQt/main.cpp

```diff
@@ -303,7 +307,7 @@ int main(int argc, char* argv[]) {
   const std::string handler_path =
       QDir(QCoreApplication::applicationDirPath()).absoluteFilePath(handler_name).toStdString();
   const std::string crash_server_url = CrashServerOptions::GetUrl();
-  const std::vector<std::string> attachments = {Path::GetLogFilePathAndCreateDir().string()};
+  const std::vector<std::string> attachments = {Path::GetLogFilePath().string()};
 
   CrashHandler crash_handler(dump_path, handler_path, crash_server_url, attachments);
 #endif  // ORBIT_CRASH_HANDLING
```

Comment:
Add a test for TryRemoveOldLogFiles?

Reply:
I split `TryRemoveOldLogFiles` into three methods, i.e., `ListFiles`, `GetLogFileLifetime`, `FilterOldLogFiles` and `RemoveFiles`;  and added tests for `GetLogFileLifetime` and `FilterOldLogFiles`. Thanks for the suggestion!